src/Essentials/src/WebAuthenticator/WebAuthenticator.ios.tvos.cs

```diff
@@ -137,15 +141,15 @@ void ClearCookies()
 #if __IOS__
 			if (OperatingSystem.IsIOSVersionAtLeast(11) || OperatingSystem.IsTvOSVersionAtLeast(11))
 			{
+#pragma warning disable CA1416 // WKWebsiteDataStore.HttpCookieStore has [SupportedOSPlatform("ios11.0")]
```

Comment:
This is also not supported on tvOS as per the apple docs.

Reply:
Right, just noticed it actually called within `#if __IOS__` condition, so the  `OperatingSystem.IsTvOSVersionAtLeast(11)` is redundant, plus causing the warning, removing that condition